massa-bootstrap/src/lib.rs

```diff
@@ -204,7 +216,7 @@ async fn get_state_internal(
         pos: Some(pos),
         graph: Some(graph),
         compensation_millis,
-        peers: Some(peers),
+        peers: peers.0.into_iter().map(|ip| (ip, false)).collect(),
```

Comment:
That can be hidden behind a `.into()` but it looks good enough for me. For the next time, you can close that comment after reading.

Reply:
Really I'd rather get rid of the BootstrapPeers struct, as it's confusinf